flowzone.yml

```diff
@@ -820,6 +820,23 @@ jobs:
             exit 1
           fi
 
+      # if auto_merge is enabled at the repository level, configure PR auto-merge
+      - name: Enable PR auto-merge
+        if: |
+          github.event.pull_request.draft == false &&
+          needs.event_types.outputs.pr == 'true' &&
```

Comment:
```suggestion
```
I don't think you need to check if it's a pr, since `do_draft` is only true for PRs. It doesn't hurt, but probably redundant.